### DISCUSSION TITLE
Does Lawnchair send data to Google products?

### DISCUSSION THREAD

**Discussion by markcellus** [LawnchairLauncher/lawnchair#2998] [Q&A] [answered]
I saw on Reddit where [a user implies that Lawnchair may send data to Google products](https://www.reddit.com/r/CalyxOS/comments/xu6izu/why_doesnt_calyx_use_lawnchair/iquxoyx/).  

> [Lawnchair] has proprietary blobs like the Google Feed
> https://f-droid.org/en/packages/ch.deletescape.lawnchair.plah/ 
> It's mentioned on f-droid

But I've never had reason to believe this to be true. Can anyone confirm this?

**Comment by SuperDragonXD** [comment, marked as answer]
Nope. Lawnchair 2 (or 1?) doesn't send data to Google products.

The proprietary blobs in V2 are basically the Google Feed intergration and Sesame Search intergation, in which Lawnchair communicates with those proprietary apps (Google and Sesame).

Also proprietary blobs doesn't mean that it's sending data to Google.
